go.mod

```diff
@@ -2,11 +2,6 @@ module github.com/carousell/Orion
 
 go 1.17
 
-replace (
-	git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
```

Comment:
As `git.apache.org/thrift.git` is not even in direct or indirect dependency listed in this go.mod, it doesn't effectively affect the result of dependency resolution. The replace is no effect, can be safely removed.